libs/dh/metering-point/feature-search/src/lib/form/dh-metering-point-search-form.component.spec.ts

```diff
@@ -0,0 +1,213 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { render, screen, fireEvent, waitFor } from '@testing-library/angular';
+import userEvent from '@testing-library/user-event';
+
+import { en as translations } from '@energinet-datahub/dh/globalization/assets-localization';
```

Comment:
suggestion: rename the variable to make it more clear which translations are used.

```suggestion
import { en as Enranslations } from '@energinet-datahub/dh/globalization/assets-localization';
```